actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   Deprecate `config.static_cache_control` in favor of
```

Comment:
Feels like this should be in the Railties CHANGELOG file. I can change it at #22173.